charts/signoz/templates/schema-migrator/migrations-init.yaml

```diff
@@ -58,6 +58,9 @@ spec:
           args:
             - "--dsn"
             - "tcp://{{ include "schemamigrator.url" . }}?username=$(CLICKHOUSE_USER)&password=$(CLICKHOUSE_PASSWORD)"
+            {{- if .Values.schemaMigrator.enableReplication }}
+            - "--replication"
```

Comment:
Is this enough or does it have to be explicit `--replication=true`

Reply:
That is enough.